src/native.rs

```diff
@@ -79,6 +94,10 @@ impl HasContext for Context {
     type UniformLocation = native_gl::GLuint;
     type TransformFeedback = native_gl::GLuint;
 
+    fn get_supported_extensions(&self) -> HashSet<String> {
+        self.extensions.clone()
```

Comment:
Could we return reference from the context here instead to avoid a `clone`? It might be convenient for callers to do `get_supported.extensions().contains("extension")` inline instead of saving the extension list themselves. On the web backend we'd also store the extension list in the context like we do on native.

Reply:
Fixed, now both implementations grab the list at initialization and expose a reference.